src/v0/destinations/salesforce/transform.js

```diff
@@ -106,24 +109,30 @@ async function getSaleforceIdForRecord(
   identifierType,
   identifierValue,
   destination,
+  authorizationFlow,
 ) {
   const objSearchUrl = `${authorizationData.instanceUrl}/services/data/v${SF_API_VERSION}/parameterizedSearch/?q=${identifierValue}&sobject=${objectType}&in=${identifierType}&${objectType}.fields=id,${identifierType}`;
+  const finalHeader =
+    authorizationFlow === OAUTH
+      ? { Authorization: `Bearer ${authorizationData.token}` }
+      : { Authorization: authorizationData.token };
   const { processedResponse: processedsfSearchResponse } = await handleHttpRequest(
     'get',
     objSearchUrl,
     {
-      headers: { Authorization: authorizationData.token },
+      headers: finalHeader,
```

Comment:
Can't we use `getAuthHeader({ authorizationFlow, authorizationData })` here ?

Reply:
done